radiative_profiles.yml

```diff
@@ -6,6 +6,13 @@ sources:
         description: Atmospheric radiative profiles from dropsondes and radiosondes published in Albright et al., 2020.
         driver: opendap
         args:
-            urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/PRODUCTS/RADIATIVE-PROFILES/rad_profiles.nc
+            urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/PRODUCTS/RADIATIVE-PROFILES/rad_profiles_{{version}}.nc
             auth: null
             chunks: null
+            engine: pydap
```

Comment:
I am wondering why you chose `pydap` as the engine. While this is the default, we usually had bad experience when using it.

Reply:
I actually cannot remember. It seems to work with both engines. I've deleted the engine key.